src/coreclr/tests/helixpublishwitharcade.proj

```diff
@@ -37,8 +37,13 @@
         BundledNETCoreAppPackageVersion=$(BundledNETCoreAppPackageVersion);
         HelixRuntimeRid=$(HelixRuntimeRid)
       </_PropertiesToPass>
-    </PropertyGroup>
 
+      
+      <_PropertiesToPass Condition="'$(TargetArchitecture)' == 'wasm' ">
+        $(_PropertiesToPass);IncludeDotNetCli=true;DotNetCliRuntime=linux-x64
```

Comment:
Do we still need my hack at line 94 with this change? Can we at least use the same RID in both places?